trex-config/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM quay.io/centos/centos:stream9
+
+ARG TREX_VERSION=2.87
+ENV TREX_VERSION ${TREX_VERSION}
+
+# install requirements
+RUN dnf -y install --nodocs git wget procps python3 vim python3-pip pciutils gettext https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm && dnf clean all
+RUN dnf install -y --nodocs hostname iproute net-tools ethtool nmap iputils perf numactl sysstat htop rdma-core-devel libibverbs libibverbs-devel net-tools && dnf clean all
+
+# install trex server
+WORKDIR /opt/
+RUN wget --no-check-certificate https://trex-tgn.cisco.com/trex/release/v${TREX_VERSION}.tar.gz && \
+    tar -xzf v${TREX_VERSION}.tar.gz && \
+    mv v${TREX_VERSION} trex && \
+    rm v${TREX_VERSION}.tar.gz
```

Comment:
I don't see a build here, why is `git` required?
Why development libraries are required? (rdma-core-devel, libibverbs-devel)

Reply:
>I don't see a build here, why is git required?

I'm not sure actually, will try to remove it

> Why development libraries are required? (rdma-core-devel, libibverbs-devel)

these are needed for the trex application.

Reply:
removed git, the devel ones, htop and the rpm one (was required for hop)